nf_core/modules/info.py

```diff
@@ -11,12 +11,10 @@
 from rich.table import Table
 from rich.text import Text
 
-from .module_utils import (
-    get_installed_modules,
-    get_module_git_log,
-    get_repo_type,
-    module_exist_in_repo,
-)
+from .module_utils import get_installed_modules
+from .module_utils import get_module_git_log
+from .module_utils import get_repo_type
+from .module_utils import module_exist_in_repo
```

Comment:
oof, @fabianegli is there a reason you wanted the one-import-per-line thing? Personally I find this much harder to read than the style above it..

Reply:
You'll quickly adapt to the style. I also found it strange in the beginning. If you are looking for an imported entity, it is faster because the imported objects align, reading the "sentenced" imports takes me longer to parse. To some extent I agree it is an acquired taste. The second point is that the diffs in PR become much cleaner when imports change. That is an objective win - I think.

Reply:
It takes a bit longer to write the import statements this way, but they are few compared to the other code and it is still possible to write them with commas and have isort fix that after the fact.

Reply:
Sorry, that was unclear on my part. Yes I definitely agree that having one imported name per line is no bad thing - I was talking about the change in the one commit when the setting was applied rather than no isort -> isort.

So they were already on their own lines before applying this setting (see the diff of [this commit](https://github.com/nf-core/tools/pull/1538/commits/a8bebd830afdc4648e0061153fe1cabc40fdf8d8#diff-00793225dafd0d24f0cb3afc14c3bb0954ad011db6af64b1e1ab683619a312b8) rather than the whole PR) - so the diffs were already clean / names aligned etc. Just a lot less verbose without the repeated namespace and (in my opinion) easier to read.. 🤔 I was wondering if I was missing a reasoning for this more verbose style.

Reply:
Aaah. I get it now. This makes it immediately clear where an import comes from. Imagine seeing a diff with and added line that starts with four spaces and then some name. (Maybe also depending on the UI) It is not immediately clear where this imported name comes from. This way it is always immediately visible. If there were no Version Control Systems, I doubt I would favor the single line imports.

Reply:
There are soooo many [options](https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html)! And the single line is what I think is best for Git, the one that was previously used here is "3 - Vertical Hanging Indent". I think the latter visually matches the standard black formatting.

Reply:
> There are soooo many [options](https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html)!

🤯 

> I think the latter visually matches the standard black formatting.

Ok that's probably why I like it 😆